test/framework/systemtools.py

```diff
@@ -454,7 +454,7 @@ def test_cpu_speed_native(self):
         """Test getting CPU speed."""
         cpu_speed = get_cpu_speed()
         self.assertTrue(isinstance(cpu_speed, float) or cpu_speed is None)
```

Comment:
Shouldn't this one also get the same change in testing order as below?

Reply:
No, that's fine. `isinstance(cpu_speed, float)` will be `False` if `cpu_speed = None`, so it'll go to the 2nd part of the `or`.

But `cpu_speed > 0.0` will fail hard if `cpu_speed = 0` in Python 3:

```
$ python3 -O -m test.framework.systemtools
.....................E.............................
======================================================================
ERROR: test_cpu_speed_native (__main__.SystemToolsTest)
Test getting CPU speed.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/shared/home/boegel/easybuild/easybuild-framework/test/framework/systemtools.py", line 457, in test_cpu_speed_native
    self.assertTrue(cpu_speed > 0.0 or cpu_speed is None)
TypeError: '>' not supported between instances of 'NoneType' and 'float'
```